sfepy/discrete/fem/mappings.py

```diff
@@ -5,11 +5,122 @@
 
 from sfepy import Config
 from sfepy.base.base import get_default, output
-from sfepy.base.mem_usage import raise_if_too_large
```

Comment:
I would like to preserve this. It allows an early failure when memory will not be sufficient for term evaluations -> infinite swapping.

Reply:
OK, I thought it was only related to allocation i C. And now nothing is allocated in C.